src/modules/backend/task.js

```diff
@@ -0,0 +1,68 @@
+/* eslint-disable import/no-mutable-exports */
+
+let tasks = [];
+
+const loadTasks = () => {
+  let load = JSON.parse(localStorage.getItem('tasks'));
+
+  if (load == null) {
+    load = [];
+  }
+
+  tasks = load;
+
+  return tasks;
+};
+
+const clearTasks = () => {
+  tasks = [];
+};
+
+const addTask = (description, completed, index) => {
```

Comment:
Good job adding the addTask method, I think you still need to add the edit and delete method as mentioned in the project requirement.

Reply:
The edit and delete methods use already existing code, or otherwise handle the frontend. It would be bad practice to repeat or mix front end and backend